kong/plugins/statsd/statsd_logger.lua

```diff
@@ -17,6 +17,8 @@ function statsd_mt:new(conf)
   end
   
   local statsd = {
+    -- FIXME: Schema migration may be better, but converting original json is not easy.
+    prefix = conf.prefix or "kong.",
```

Comment:
yes it's not hard. I've made some other migration during my own development, but only for DB schema changes via sql. let me see how to upgrade a json content there.